__tests__/app.test.js

```diff
@@ -53,3 +52,39 @@ describe("/api", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  test("GET: 200 sends an article matching the article_id.", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .expect(200)
+      .then(({ body }) => {
+        const article = body.article;
+        expect(article.article_id).toBe(2);
+      });
+  });
+  test("Article should be of the correct format.", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .then(({ body }) => {
+        const article = body.article;
+        expect(article).toHaveProperty("author");
+        expect(article).toHaveProperty("title");
+        expect(article).toHaveProperty("article_id");
+        expect(article).toHaveProperty("body");
+        expect(article).toHaveProperty("topic");
+        expect(article).toHaveProperty("created_at");
+        expect(article).toHaveProperty("votes");
+        expect(article).toHaveProperty("article_img_url");
+      });
+  });
+
+  test("GET 404 Article Not Found", () => {
```

Comment:
You have a 404 test here which is fab, but we are also going to need a 400 test for when someone passes in an article_id which is invalid - /api/articles/cheese - for example. 

Postgresql will give you back an error here so you can use that building your error handler for this test